example/image-classification/train_mnist.py

```diff
@@ -59,9 +59,8 @@ def get_mnist_iter(args, kv):
         network        = 'mlp',
         # train
         gpus           = None,
-        batch_size      = 64,
-        disp_batches = 100,
-        num_epochs     = 20,
```

Comment:
I removed the --num-epochs argument because it gets set in add_fit_args() where a default is eset.
Here I replaced the num_epochs.